.github/workflows/health.yaml

```diff
@@ -55,17 +55,36 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
+      - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
+        with:
+          ref: ${{ github.event.pull_request.base.ref }}
+          path: .coverage_base
+          
       - uses: dart-lang/setup-dart@d6a63dab3335f427404425de0fbfed4686d93c4f
         with:
           sdk: ${{ inputs.sdk }}
 
+      - name: Install coverage
+        run: dart pub global activate coverage
+
+      - name: Get coverage for base and head of branch
+        run: find . -name "pubspec.yaml" -exec dirname {} \; | while read -r line; do (set -x; cd "$line"; dart pub get; if [ -d test ]; then dart pub global run coverage:test_with_coverage; fi); done
```

Comment:
I _think_ that this is:
- traversing into the `.` and `.coverage_base` dirs (cloned from above)
- finding things that reasonably look like packages (has a pubspec; has tests)
- running tests w/ coverage; leaving the lconv artifacts behind in the various packages

Having the logic in this bash line will make it tough to handle the various types of repos we have. I think we should evolve this (in this PR or a later one) to living in dart code. Perhaps w/ an entry-point like `dart pub run firehose:collect_coverage <dir>`?